internal/backends/winit/wasm_input_helper.rs

```diff
@@ -67,6 +70,58 @@ impl WasmInputHelper {
 
         let shared_state = Rc::new(RefCell::new(WasmInputState::default()));
 
+        let win = window_adapter.clone();
```

Comment:
I suppose most of this code needs to be guarded with 
`#[cfg(web_sys_unstable_apis)]`